File: app/src/main/java/com/ptithcm/thuan6420/basecleanarchitecture/ui/home/MainActivity.java
package com.ptithcm.thuan6420.basecleanarchitecture.ui.home;

import android.os.Bundle;

import androidx.appcompat.app.AppCompatActivity;

import com.ptithcm.thuan6420.basecleanarchitecture.R;
import com.ptithcm.thuan6420.basecleanarchitecture.databinding.ActivityMainBinding;

public class MainActivity extends AppCompatActivity {
    private ActivityMainBinding binding;

    @Override
    protected void onCreate(Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);
        binding = ActivityMainBinding.inflate(getLayoutInflater());
        setContentView(binding.getRoot());
    }
}

File: app/src/main/java/com/ptithcm/thuan6420/basecleanarchitecture/ui/home/view/MainActivity.kt
package com.ptithcm.thuan6420.basecleanarchitecture.ui.home.view

import androidx.appcompat.app.AppCompatActivity
import android.os.Bundle
import com.ptithcm.thuan6420.basecleanarchitecture.databinding.ActivityMainBinding
import com.ptithcm.thuan6420.basecleanarchitecture.ui.home.presenter.HomeContact

class MainActivity : AppCompatActivity(), HomeContact.ViewInterface {
    private var binding: ActivityMainBinding? = null
    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)
        binding = ActivityMainBinding.inflate(
            layoutInflater
        )
        setContentView(binding!!.root)
    }

    override fun onLoading(status: Boolean) {

    }
}